libs/logger/src/lib/logger.ts

```diff
@@ -33,18 +32,9 @@ export function initializeLogger(
 function initializeTestLogger() {
   PostyBirbDirectories.initializeDirectories();
 
-  const transport = new DailyRotateFile({
-    filename: 'postybirb-%DATE%.log',
-    datePattern: 'YYYY-MM-DD',
-    zippedArchive: true,
-    maxSize: '20m',
-    maxFiles: '14d',
-    dirname: PostyBirbDirectories.LOGS_DIRECTORY,
-  });
```

Comment:
It was making tests fail because it tried to write and got EACCESS. Not sure why tests need file logger anyway